src/components/Accordion/AccordionItem.tsx

```diff
@@ -0,0 +1,83 @@
+import { px } from '../../utils';
+import plusIcon from '../../assets/plus.svg';
+import minusIcon from '../../assets/minus.svg';
+import lockIcon from '../../assets/lock.svg';
+import lockBlueIcon from '../../assets/lockBlue.svg';
+import { useState } from 'react';
+import classnames from 'classnames';
+
+export interface AccordionItemProps extends React.HTMLAttributes<HTMLDivElement> {
+  id?: string;
+  key?: string;
+  isLastItem?: boolean;
+  isLocked: boolean;
+  isLockedVariation?: boolean;
+  // eslint-disable-next-line  @typescript-eslint/no-explicit-any
+  lockedContent?: any;
+  variation: 'lg' | 'sm';
+  label: string;
+  // eslint-disable-next-line  @typescript-eslint/no-explicit-any
+  content?: any;
+  hasTransition?: boolean;
```

Comment:
When writing out the interface you should have comments above each one. This is what gets populated in our docs for storybook. You can look at the [grid component](https://github.com/PhillipsAuctionHouse/seldon/blob/ef7ee39ff5f7d8fe9286ba4b0bcb0cd9bcbe9556/src/components/Grid/Grid.tsx#L5) for example.

I do believe it needs to be a multiline comment e.g.

```
/**
* Your comment becomes documentation for the prop
*/